src/tagify.js

```diff
@@ -173,7 +189,8 @@ Tagify.prototype = {
             fuzzySearch   : true,
             highlightFirst: false,  // highlights first-matched item in the list
             closeOnSelect : true,   // closes the dropdown after selecting an item, if `enabled:0` (which means always show dropdown)
-            position      : 'all'   // 'manual' / 'text' / 'all'
+            position      : 'all',  // 'manual' / 'text' / 'all',
+            unaccent      : true    // needs a polyfill to work with IE. https://github.com/andyearnshaw/Intl.js/
```

Comment:
Thank you very much for the pull request! I appreciate the help on this and have a few thoughts:

The name of this *setting* isn't clear enough. What is unaccented? the rendered results?
I would be more explicit and name it something like `accentedSearch` because there's already a setting called `fuzzySearch` which also affects how the whitelist gets filtered.

Also I've checked the `Intl.js` you've linked to, and I didn't see it polyfills `String.normalize` method. Also it looks very bloated and most irrelevant to include for such tiny polyfill.

I maintain my own polyfill file `tagify.polyfills.js` under `./src` folder, and I think it's best to find a good polyfill and put it there. I was many and each one is different from the other, so it's a rigorous task picking the best one of them, but nonetheless important so Tagify user's won't experience potential bugs.

What are your thoughts?

Reply:
I found this lib [unorm](https://github.com/walling/unorm). It is used as polyfill by this project here: [Financial-Times/polyfill-library](https://github.com/Financial-Times/polyfill-library/tree/bcc4bc3ef89f3935755d209bea641bfdee6efa72/polyfills/String/prototype/normalize)
But I'm not sure how to extract it to `tagify.polyfills.js`, if you can help me, I appreciate.

About setting name, you're right. It's better to change to `accentedSearch` or `unaccentedSearch`